README.md

```diff
@@ -43,6 +43,7 @@ The following commands are available:
 - `/ipfabric end-to-end-path [src-ip] [dst-ip] [src-port] [dst-port] [protocol]`: Execute end-to-end path simulation between source and target IP address.
 - `/ipfabric routing [device] [protocol] [filter-opt]`: Get routing information for a device.
 - `/ipfabric wireless [option] [ssid]`: Get wireless information by client or ssid.
+- `/ipfabric diff [snapshot_id] [table] [view]`: Get either a summary or detailed diff of table between the set snapshot and snapshot_id.
```

Comment:
from the description, it's not clear to me _how_ you get  the summary or the detailed diff. Also, if it's a diff of a table, I would make it more explicit. What about `/ipfabric table_diff`?

Reply:
Maybe `/ipfabric table-diff` for consistency?